samples/caf_sensor_manager/cpunet/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(net)
+
+#Include application application event headers
+zephyr_library_include_directories(../src/events)
+
+target_sources(app PRIVATE
+  src/main.c
+  ../src/modules/workload_sim.c
```

Comment:
...?

I don't get it. This should execute on application core. Also why adding this code is not a property of module and module is not enabled in kconfig for this core?